pages/search/search.jsx

```diff
@@ -9,6 +9,12 @@ import Service from '../../js/service.js';
 import ProjectLoader from '../../components/project-loader/project-loader.jsx';
 
 const DEFALUT_MODERATION_FILTER = `Pending`;
+const DEFALUT_FEATURED_FILTER = `---`;
```

Comment:
Yea I was thinking if we should give moderators more options to filter but now I think about it the not-featured option might not be used much.